web/src/pages/Route/components/DebugViews/AuthenticationView.tsx

```diff
@@ -0,0 +1,127 @@
+  const { formatMessage } = useIntl();
+  const [authType, setAuthType] = useState('none');
+
+  const getAuthFormItems = () => {
+    switch (props.form.getFieldValue('authType')) {
+      case 'basic-auth':
+        return (
+          <>
+            <Form.Item
+              label={formatMessage({ id: 'page.route.form.itemLabel.username' })}
+              name="username"
+              rules={[
+                {
+                  required: true,
+                  message: `${formatMessage({
+                    id: 'component.global.pleaseEnter',
+                  })}${formatMessage({ id: 'page.route.form.itemLabel.username' })}`,
+                },
+              ]}
+            >
+              <Input />
+            </Form.Item>
+            <Form.Item
+              label={formatMessage({ id: 'page.route.form.itemLabel.password' })}
+              name="password"
+              rules={[
+                {
+                  required: true,
+                  message: `${formatMessage({
+                    id: 'component.global.pleaseEnter',
+                  })}${formatMessage({ id: 'page.route.form.itemLabel.password' })}`,
+                },
+              ]}
+            >
+              <Input.Password />
+            </Form.Item>
+          </>
+        );
+      case 'jwt-auth':
+        return (
+          <Form.Item
+            label={formatMessage({ id: 'page.route.form.itemLabel.token' })}
+            name="Authorization"
+            rules={[
+              {
+                required: true,
+                message: `${formatMessage({ id: 'component.global.pleaseEnter' })}${formatMessage({
+                  id: 'page.route.form.itemLabel.token',
+                })}`,
+              },
+            ]}
+          >
+            <Input />
+          </Form.Item>
+        );
+      case 'key-auth':
+        return (
+          <Form.Item
+            label={formatMessage({ id: 'page.route.form.itemLabel.apikey' })}
+            name="apikey"
+            rules={[
+              {
+                required: true,
+                message: `${formatMessage({ id: 'component.global.pleaseEnter' })}${formatMessage({
+                  id: 'page.route.form.itemLabel.apikey',
+                })}`,
+              },
+            ]}
+          >
+            <Input />
+          </Form.Item>
+        );
+      default:
+        return <div>{formatMessage({ id: 'page.route.debugWithoutAuth' })}</div>;
+    }
+  };
+
+  return (
+    <Card>
+      <Form name="authForm" form={props.form}>
+        <div className={styles.authForm}>
+          <Form.Item name="authType">
+            <Radio.Group
+              defaultValue={authType}
+              onChange={(event) => {
+                const currentValue = event.target.value;
+                setAuthType(currentValue);
+                props.form.setFieldsValue({ autyType: currentValue });
+              }}
+            >
+              <Radio value="none">none</Radio>
```

Comment:
use `None` as label would be better

```suggestion
              <Radio value="none">None</Radio>
```

Reply:
done